traitsui/tests/editors/test_enum_editor.py

```diff
@@ -521,3 +448,47 @@ def test_list_evaluate_editor_text(self):
 
     def test_list_evaluate_editor_index(self):
         self.check_enum_index_update(get_evaluate_view("custom", mode="list"))
+
+    def check_list_enum_none_selected(self, view):
+        enum_edit = EnumModelWithNone()
+        view = View(
```

Comment:
Whatever `view` provided is overridden here, so the `test_list_enum_none_selected` and `test_list_evaluate_none_selected` are effectively identical. Perhaps we can just change this `check_list_...` to `test_...` and remove the other two?